src/Development/IDE/Spans/Common.hs

```diff
@@ -30,23 +29,22 @@ import ConLike
 import DataCon
 import Var
 import NameEnv
+import DynFlags
 
 import qualified Documentation.Haddock.Parser as H
 import qualified Documentation.Haddock.Types as H
-import Development.IDE.GHC.Compat
 import Development.IDE.GHC.Orphans ()
+import Development.IDE.GHC.Util
 
 type DocMap = NameEnv SpanDoc
 type KindMap = NameEnv TyThing
 
-showGhc :: Outputable a => a -> String
-showGhc = showPpr unsafeGlobalDynFlags
 
-showName :: Outputable a => a -> T.Text
-showName = T.pack . prettyprint
-  where
-    prettyprint x = renderWithStyle unsafeGlobalDynFlags (ppr x) style
-    style = mkUserStyle unsafeGlobalDynFlags neverQualify AllTheWay
+showGhc :: Outputable a => a -> T.Text
+showGhc = showSD . ppr
+
+showSD :: SDoc -> T.Text
+showSD = T.pack . unsafePrintSDoc
```

Comment:
This used to render with `neverQualify AllTheWay`. Why remove those?

Reply:
Same as `showSDoc`, but maybe I can re-add it for explicitness: https://hackage.haskell.org/package/ghc-8.10.1/docs/src/Outputable.html#showSDoc

Reply:
Given how much the ghc api changes across versions, my vote is on explicitness